app/gcl/associate-cloud-engineer/cloud-load-balancing-guide/page.css

```
@@ -0,0 +1,719 @@
+.cloud-load-balancing-guide-page .vz-packet.f2 {
+    offset-path: path('M 150 150 L 470 150');
+    animation-delay: 0.8s;
+}
+
+.cloud-load-balancing-guide-page .vz-packet.f3 {
+    offset-path: path('M 150 150 L 470 230');
+    animation-delay: 1.6s;
+}
+
+@keyframes vzflow {
+    from {
```

Stylelint (17.13.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)